### PR TITLE
Add the Google API client library generator

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

**Comment by saicheems** [comment]
This is the open source client library generator - I need it to generate mocks for Java and PHP.

**Comment by tcoffee-google** [comment]
Could you modify `README.md` to explain the scope of this new directory?

**Comment by saicheems** [comment]
Added a note, let me know if you want me to change it and I'll follow up.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Introduction
 
-The Discovery Artifact Manager is intended to facilitate testing, publishing, and synchronization of [Toolkit](https://github.com/googleapis/toolkit/), Discovery files from the [Discovery service](https://developers.google.com/discovery/), and [Discovery-based Google API client libraries](https://developers.google.com/discovery/libraries).
+The Discovery Artifact Manager is intended to facilitate testing, publishing, and synchronization of [Toolkit](https://github.com/googleapis/toolkit/).
```
Comment (tcoffee-google) [inline README.md]: This no longer makes sense to me ... dartman is intended to facilitate testing/publishing/sync of *all* these things with each other. Toolkit is one of the things, and should be among the list of sources copied in the repo.

**Pull request merged**